website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -0,0 +1,491 @@
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "no-code-modules",
+    "attributes": {
+      "enabled": false
+    },
+    "relationships": {
+      "registry-module": {
+        "data": {
+          "id": "mod-zyai9dwH4VPPaVuC",
+          "type": "registry-module"
+        }
+      },
+      "variable-options": {
+        "data": [
+          {
+            "id": "ncvaropt-fcHDfnZ1EGdRzFNC",
+            "type": "variable-options",
+            "attributes": {
+              "variable-name": "Linux AMIs",
+              "variable-type": "array",
+              "options": [
+                "Xenial Xerus",
+                "Trusty Tahr"
+              ]
+            }
+          },
+          {
+            "id": "ncvaropt-dZMfdh9KBcwFjyv2",
+            "type": "variable-options",
+            "attributes": {
+              "variable-name": "region",
+              "variable-type": "array",
+              "options": [
+                "eu-north-1",
+                "us-east-2",
+                "us-west-1"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/no-code-modules/nocode-9HE91XDNY3faePn2
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "nocode-9HE91XDNY3faePn2",
+    "type": "no-code-modules",
+    "attributes": {
+      "enabled": true,
+      "version-pin": "1.0.1"
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        },
+        "links": {
+          "related": "/api/v2/organizations/hashicorp"
+        }
+      },
+      "registry-module": {
+        "data": {
+          "id": "mod-2aaFrmRPZs2N9epr",
+          "type": "registry-modules"
+        },
+        "links": {
+          "related": "/api/v2/registry-modules/mod-2aaFrmRPZs2N9epr"
+        }
+      },
+      "variable-options": {
+        "data": [
+          {
+            "id": "ncvaropt-fcHDfnZ1EGdRzFNC",
+            "type": "variable-options"
+          },
+          {
+            "id": "ncvaropt-dZMfdh9KBcwFjyv2",
+            "type": "variable-options"
+          }
+        ]
+      }
+    },
+    "links": {
+      "self": "/api/v2/no-code-modules/nocode-9HE91XDNY3faePn2"
+    }
+  }
+}
+```
+
+## Read a No-Code-enabled module's properties
+
+
+`GET /no-code-modules/:id`
+
+
+Parameter | Description
+--- | ---
+`:id` | The unique identifier for a No-Code-Allowed module.
+
+Use this API to read the details of an existing No-Code-Allowed module.
+
+The unique identifier for the No-Code-Allowed module was returned in the response to [the API call that enabled the module for No-Code Provisioning](#allow-no-code-provisioning-of-a-module-for-an organization).
```

Comment:
Might need to to tweak this link if you accept the title rename suggestion above:
```suggestion
The [API call that enables for no-code provisioning for a module](#allow-no-code-provisioning-of-a-module-for-an organization) returns that module's unique identifier.
```